### DISCUSSION TITLE
Possibility to open web search in browser privacy tab

### DISCUSSION THREAD

**Discussion by MikiGrit** [Flow-Launcher/Flow.Launcher#1315] [Ideas]
Due to the privacy reasons when I search on the internet (e.g. researching for which product to buy), I'm used to do the search on the private tab (also called anonymous). Would it be possible to implement such functionality in the Flow Launcher?

**Comment by Garulf** [comment]
This is actually already a part of Flow Launcher. I'm not at my PC ATM but I'll post a screenshot as soon as I can.

**Reply by MikiGrit** [reply]
Oh, really? That's great info. Thanks, I've searched for such possibility but was unable to find any information about it.

**Comment by Garulf** [comment]
![image](https://user-images.githubusercontent.com/535299/183518475-d6e7a4b8-8a9c-4dbf-86ef-66b524009dcf.png)

![image](https://user-images.githubusercontent.com/535299/183518531-542a1333-6fda-4b80-a8b9-b75e4cf92e15.png)

**Reply by rs911** [reply]
Hi
I tried using this, but it just opens the tab in the current browser window. 
I tried using different chrome profiles 

"C:\Program Files\Google\Chrome\Application\chrome.exe" --profile-directory="Profile 1"

Even tried putting the --incognito option. But still it opens in the already open browser window.

Although it did open the private tab in MSEdge. Just not working with Chrome. Did not try firefox.
Can someone please confirm if its just me?

**Comment by z1nc0r3** [comment]
![image](https://user-images.githubusercontent.com/64279853/210127218-d5f5b293-69ad-4c36-8f2b-d444614fa15b.png)

Working fine with these settings. Open a new incognito window (if already open, new tab) for web searches.

**Reply by rs911** [reply]
Thank you, it is working without adding the browser path.
I tried adding a new browser with the path to chrome. It doesn't work with that.

**Reply by z1nc0r3** [reply]
Happy to hear that :)
